src/librustdoc/html/render/print_item.rs

```diff
@@ -1769,6 +1769,13 @@ fn document_type_layout(w: &mut Buffer, cx: &Context<'_>, ty_def_id: DefId) {
                  the type was too big.</p>"
             );
         }
+        Err(LayoutError::NormalizationFailure(_, _)) => {
+            writeln!(
+                w,
+                "<p><strong>Note:</strong> Encountered an error during type layout; \
+                the type was not normalizable.</p>"
```

Comment:
Could you change this to "the type failed to be normalized"? The current wording is a bit confusing IMO.